flexbox.html

```diff
@@ -1,16 +1,140 @@
 <!DOCTYPE html>
 <html>
-  <head>
+
+<head>
     <meta charset="UTF-8">
-  <meta name="viewport" content="width=device-width,initial-scale=1">
+    <meta name="viewport" content="width=device-width,initial-scale=1">
     <title>title</title>
-  <meta name="author" content="name">
-  <meta name="description" content="description here">
-  <meta name="keywords" content="keywords,here">
-  <link rel="stylesheet" href="./css/flexbox.css" type="text/css">
-  </head>
-  <body>
-
-  <script src="index.js" type="text/javascript"></script>
-  </body>
-</html>
+    <meta name="author" content="name">
+    <meta name="description" content="description here">
+    <meta name="keywords" content="keywords,here">
+    <link rel="stylesheet" href="./css/flexbox.css" type="text/css">
+</head>
+
+<body>
+
+    <nav>
+
+        <div class=nav-menu>
+            <ul class=nav-left>
+                <li><a href="#m_origin">Marvel's Origin</a></li>
+                <li><a href="#strongest_Heroes">Strongest Heroes</a></li>
+                <li><a href="#Marvel_Characters">Marvel Characters</a></li>
+                <li><a href="#Matchups">Matchups</a></li>
+                <li><a href="#Comics">Comics</a></li>
+                <li style="float:right"><img src="img/arod1.png" class="avatar-img"></li>
+                <li style="float:right"><a class="nav-right" href="/Users/codethedream/lessons/Sunrise-phoenix/front-end-1-section-1-alexrz07/index.html">Marvel (float)</a></li>
```

Comment:
href="/Users/codethedream/lessons/Sunrise-phoenix/front-end-1-section-1-alexrz07/index.html">

may work in local but doesnt roote in other plotforms. It could change to "index.html"